src/index.js

```diff
@@ -1,14 +1,33 @@
-import _ from 'lodash';
 import './style.css';
 
- function component() {
-   const element = document.createElement('div');
+const toDoTasks = [
+  {
+    description: 'Exercise for 30 min',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'Get Ready and breakfast',
+    completed: true,
+    index: 1,
+  },
+  {
+    description: 'Study for 1 hour',
+    completed: false,
+    index: 2,
+  },
+];
 
-   // Lodash, now imported by this script
-   element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-  element.classList.add('hello');
+const List = document.querySelector('.item-list');
 
-   return element;
- }
+const createList = () => {
+  toDoTasks.forEach((task) => {
+    const element = document.createElement('li');
+    element.innerHTML = `<div class="left"><i class="far fa-square"></i> ${task.description} </div>
+    <i class="fa fa-ellipsis-v"></i>`;
+    element.classList.add('rows');
+    List.appendChild(element);
+  });
```

Comment:
Please make sure that the tasks appear in order of the index values for each task. They are currently being rendered in the order of the Array index. This feature would be helpful later when deleting tasks as they will readjust themselves using the `index` value of the task.